Source/ReactiveCore/MoyaError.swift

```diff
@@ -9,9 +9,10 @@ public enum MoyaError: ErrorType {
     case Underlying(ErrorType)
 }
 
-// If RACSignal support gets removed we can also remove everything under here
+@available(*, deprecated, message="This will be removed when ReactiveCocoa 4 becomes final. Please visit https://github.com/Moya/Moya/issues/298 for more information.")
 public let MoyaErrorDomain = "Moya"
 
+@available(*, deprecated, message="This will be removed when ReactiveCocoa 4 becomes final. Please visit https://github.com/Moya/Moya/issues/298 for more information.")
 public enum MoyaErrorCode: Int {
```

Comment:
I should think that, in fact, we can merely rename these types, remove the domain and take away the underlying `Int` – these types are still useful and ought to remain.